docs/eventing/samples/writing-event-source-easy-way/README.md

```diff
@@ -0,0 +1,298 @@
+create event sources. The way in that tutorial is to create an independent event source that has its own CRD.
+
+In this tutorial though, you will build an event source in Javascript and use it with
+[ContainerSource](../../../eventing/sources/README.md#meta-sources) and / or [SinkBinding](../../../eventing/sources/README.md#meta-sources).
+
+[ContainerSource](../../../eventing/sources/README.md#meta-sources) is an easy way to turn any dispatcher container into an Event Source.
+Similarly, another option is using [SinkBinding](../../../eventing/sources/README.md#meta-sources)
+which provides a framework for injecting environment variables into any Kubernetes resource which has a `spec.template` that looks like a Pod (aka PodSpecable).
+
+Code for this tutorial is available [here](./).
+
+# Bootstrapping
+
+Create the project and add the dependencies:
+```bash
+npm init
+npm install cloudevents-sdk --save
+```
+
+## Making use of ContainerSource
+
+`ContainerSource` and `SinkBinding` both work by injecting environment variables to an application.
+Injected environment variables at minimum contain the URL of a sink that will receive events.
+
+Following example emits an event to the sink every 1000 milliseconds.
+The sink URL to post the events will be made available to the application via the `K_SINK` environment variable by `ContainerSource`.
+
+```javascript
+// File - index.js
+
+const v1 = require("cloudevents-sdk/v1");
+
+let sinkUrl = process.env['K_SINK'];
+
+console.log("Sink URL is " + sinkUrl);
+
+let config = {
+    method: "POST",
+    url: sinkUrl
+};
+
+// The binding instance
+let binding = new v1.BinaryHTTPEmitter(config);
+
+let eventIndex = 0;
+setInterval(function () {
+    console.log("Emitting event #" + ++eventIndex);
+
+    // create the event
+    let myevent = v1.event()
+        .id('your-event-id')
+        .type("your.event.source.type")
+        .source("urn:event:from:your-api/resource/123")
+        .dataContentType("application/json")
+        .data({"hello": "World " + eventIndex});
+
+    // Emit the event
+    binding.emit(myevent)
+        .then(response => {
+            // Treat the response
+            console.log("Event posted successfully");
+            console.log(response.data);
+        })
+        .catch(err => {
+            // Deal with errors
+            console.log("Error during event post");
+            console.error(err);
+        });
+}, 1000);
+```
+
+```dockerfile
+# File - Dockerfile
+
+FROM node:10
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm install
+COPY . .
+EXPOSE 8080
+CMD [ "node", "index.js" ]
+
+```
+
+Build and push the image:
+```bash
+docker build . -t path/to/image/registry/node-knative-heartbeat-source:v1
+docker push path/to/image/registry/node-knative-heartbeat-source:v1
+```
+
+Create the event display service which simply logs any cloudevents posted to it.
```

Comment:
Do you want to use `kn` to create this service, instead of needing to fill in YAML?

Actually, that may not be necessary if this is always a fixed (published) image. Is `event-display` part of the released container images?

Reply:
`event-display` is part of the released container images, yep.